### PR TITLE
Add autofocus to form inputs

### PR DESCRIPTION
**Pull request by katedee**
From the usability feedback, there was some issue with the inputs not having focus. People would type what was on the letter provided, but look up, and see a blank input and be confused.

To solve this, let's autofocus on the first input, or on errors, the first input in the list of errors.

This is a start, as I haven't tackled all the forms we have in the app.

WIP because I need to add tests

### CODE REVIEW

File: views/login/code.pug [hunk 1]
```diff
@@ -17,7 +17,7 @@ block content
         span.cra-form-message #{__('For Example:')} A5G98S4K1
         if errors
           +validationMessage(errors.code.msg, 'code')
-        input.w-3-4#code(name='code', type='text', value=data.code, autocomplete='off' aria-describedby=(errors && errors.code ? 'code-error' : false))
+        input.w-3-4#code(name='code', type='text', autofocus, value=data.code, autocomplete='off' aria-describedby=(errors && errors.code ? 'code-error' : false))
```
Comment (pcraig3) [inline views/login/code.pug]: Could we use the text-input mixin here instead of keeping it as-is?

File: views/personal/address-edit.pug [hunk 2]
```diff
@@ -21,7 +21,7 @@ block content
       h2 #{__('Your mailing address')}
 
       div
-        +textInput('Street number', ['w-1-4','d--ib'])(class='w-3-4', value=streetNumber, name='streetNumber')
+        +textInput('Street number', ['w-1-4','d--ib'])(class='w-3-4', value=streetNumber, name='streetNumber' autofocus=(!errors || errors && firstError.param === 'streetNumber' ? true : false))
```
Comment (pcraig3) [inline views/personal/address-edit.pug]: Can we put the `firstError` var in this template itself since it's the only time we use it?

Comment (katedee) [reply views/personal/address-edit.pug]: definitely could. I was thinking about in the future when there might be more forms with multiple inputs. But you're right that for now it's just this page

File: views/login/code.pug (hunk 1 above)
Comment (katedee) [reply views/login/code.pug]: sure thing

File: views/personal/address-edit.pug (hunk 2 above)
Comment (pcraig3) [reply views/personal/address-edit.pug]: I think maybe you deleted 'firstError' 🤔

**Pull request merged**